tests/test_app.py

```diff
@@ -216,6 +216,12 @@ def test_branch_regexp():
             assert bot.config.branch_regexp == re.compile('foo.*bar')
 
 
+def test_source_branch_regexp():
```

Comment:
I think what should be tested is that by default all source branches are processed, but if a filter is explicitly specified only source branches that match is get merged.